spec/lib/deep_delete_spec.rb

```diff
@@ -0,0 +1,67 @@
+# default_ansible_object (resource)?
+
+RSpec.describe DeepDelete do
+  let(:ems) { FactoryBot.create(:ems_container) }
+  let(:ems_other) { FactoryBot.create(:ems_container) }
+
+  # silence the output
+  before do
+    allow(DeepDelete).to receive(:puts)
+    allow_any_instance_of(DeepDelete).to receive(:puts)
```

Comment:
What is outputting anything anymore?  I don't see puts calls above so maybe this can be removed?  I am wary about a library that has puts anyway (otherwise it should be in the tools dir).